script.js

```diff
@@ -0,0 +1,66 @@
+const bookList = document.querySelector('.book-list');
+const form = document.querySelector('#book-form');
+
+// book collection
+let collection = [];
+
+/**
+ * Renders all the books in the array to the UI
+ * @param {string, string, number} title, author, bookId
+ */function addToBookList(title, author, bookId) {
+  // create all necessary variables
+  const bookTitle = document.createElement('p');
+  bookTitle.innerText = title;
+  const bookAuthor = document.createElement('p');
+  bookAuthor.innerText = author;
+  const removeBtn = document.createElement('button');
+  removeBtn.innerText = 'Remove';
+  const hr = document.createElement('hr');
+
+  // create container and append all these
+  const bookContainer = document.createElement('div');
+  bookContainer.appendChild(bookTitle);
+  bookContainer.appendChild(bookAuthor);
+  bookContainer.appendChild(removeBtn);
+  bookContainer.appendChild(hr);
```

Comment:
[Optional] In order to make your code cleaner you can append multiple child elements to the parent element in one line.
`
bookContainer.append(firstChild, secondChild, thirdChild, ...);
`

Kindly check for other occurrences 👍